src/shared/lib/comparison/ComparisonStore.ts

```diff
@@ -1035,6 +1064,34 @@ export default class ComparisonStore {
         []
     );
 
+    readonly survivalClinicalAttributesPrefix = remoteData({
+        await: () => [this.activeStudiesClinicalAttributes],
+        invoke: () => {
+            const attributes = getSurvivalAttributes(
+                this.activeStudiesClinicalAttributes.result!
+            );
+            let attributesPrefix: string[] = [];
+            // get paired attributes
+            attributes.forEach(attribute => {
+                let prefix = attribute.substring(
+                    0,
+                    attribute.indexOf('_STATUS')
+                );
+                if (!attributesPrefix.includes(prefix)) {
+                    if (attributes.includes(`${prefix}_MONTHS`)) {
+                        attributesPrefix.push(prefix);
+                    }
+                }
+            });
+            // change prefix order based on priority
+            return Promise.resolve(
+                _.sortBy(attributesPrefix, prefix => {
+                    return plotsPriority[prefix] || 999;
```

Comment:
999 is a bit of a magic number here.  make a constant with a name that explains what it is.  like, MAX_PRIORITY or something